USAGE.md

```diff
@@ -1,39 +1,124 @@
 # How to use avTranscoder
 
-Check out applications contained in the project to see examples of how to use the library in C++, Java or Python.
+## When to use avTranscoder
```

Comment:
I don't know, because if we do so you should check each 'avTranscoder' word in the doc... And maintain that.